pkg/kubectl/resource_printer.go

```diff
@@ -815,13 +815,18 @@ func printPersistentVolumeClaim(pvc *api.PersistentVolumeClaim, w io.Writer, wit
 		}
 	}
 
-	if withNamespace {
-		if _, err := fmt.Fprintf(w, "%s\t", namespace); err != nil {
-			return err
-		}
+	labels := formatLabels(pvc.Labels)
+	phase := pvc.Status.Phase
+	storage := pvc.Spec.Resources.Requests[api.ResourceStorage]
+	capacity := int64(0)
+	accessModes := ""
+	if pvc.Spec.VolumeName != "" {
+		accessModes = volume.GetAccessModesAsString(pvc.Status.AccessModes)
+		storage = pvc.Status.Capacity[api.ResourceStorage]
+		capacity = storage.Value()
```

Comment:
I missed the pretty-print string version somehow and went for the int value.  

I added the string here and also to the PV resource_printer and describer.